cmake/schemes/url_sha1_avahi_autotools.cmake.in

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2013, 2015 Ruslan Baratov, Alexandre Pretyman
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/modules")
+
+include("@HUNTER_SELF@/cmake/Hunter")
+
+include(hunter_fatal_error)
+include(hunter_status_debug)
+include(hunter_test_string_not_empty)
+include(hunter_autotools_project)
+
+hunter_status_debug("Scheme: url_sha1_autotools")
+
+if("@MSVC@")
+  hunter_fatal_error("Autotools scheme not supported with Visual Studio")
+endif()
+
+# Check preconditions
+hunter_test_string_not_empty("@HUNTER_SELF@")
+hunter_test_string_not_empty("@HUNTER_EP_NAME@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_URL@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SHA1@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_DOWNLOAD_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SOURCE_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_BUILD_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_INSTALL_PREFIX@")
+
+hunter_add_package(Expat)
+hunter_add_package(libdaemon)
+hunter_add_package(intltool)
+
+hunter_autotools_project("@HUNTER_EP_NAME@"
+    HUNTER_SELF
+      "@HUNTER_SELF@"
+    URL
+      @HUNTER_PACKAGE_URL@
+    URL_HASH
+      SHA1=@HUNTER_PACKAGE_SHA1@
+    DOWNLOAD_DIR
+      "@HUNTER_PACKAGE_DOWNLOAD_DIR@"
+    SOURCE_DIR
+      "@HUNTER_PACKAGE_SOURCE_DIR@"
+    BUILD_DIR
+      "@HUNTER_PACKAGE_BUILD_DIR@"
+    INSTALL_DIR
+      "@HUNTER_PACKAGE_INSTALL_PREFIX@"
+    PARALLEL_JOBS
+      "@HUNTER_JOBS_OPTION@"
+    EXTRA_FLAGS
+     --disable-qt3
+     --disable-qt4
+     --disable-gtk
+     --disable-gtk3
+     --disable-gdbm
+     --disable-python
+     --disable-mono
+     --disable-monodoc
+     --with-xml=expat
+     --enable-compat-libdns_sd
```

Comment:
I guess we need an extra flags here to pass a custom location of dependencies `Expat`, `libdaemon` and `intltool`. I've got next error while trying to build example:
```
cd /.../Hunter/_Base/93af25f/fc4ba94/b270781/Build/Avahi/Source && . /.../Hunter/_Base/Download/Hunter/unknown/93af25f/Unpacked/scripts/clear-all.sh && AR=/usr/bin/ar ./configure AR=/usr/bin/ar LD=/usr/bin/ld NM=/usr/bin/nm OBJCOPY=/usr/bin/objcopy OBJDUMP=/usr/bin/objdump RANLIB=/usr/bin/ranlib STRIP=/usr/bin/strip CC=/usr/bin/gcc CXX=/usr/bin/g++ CXXFLAGS=-std=c++11 --disable-qt3 --disable-qt4 --disable-gtk --disable-gtk3 --disable-gdbm --disable-python --disable-mono --disable-monodoc --with-xml=expat --enable-compat-libdns_sd --enable-shared --disable-static --prefix=/.../Hunter/_Base/93af25f/fc4ba94/b270781/Install
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
...
checking whether NLS is requested... yes
checking for intltool >= 0.35.0... ./configure: line 18800: intltool-update: command not found
```